doc/releases/changelog-dev.md

```diff
@@ -876,6 +876,9 @@
 
 <h3>Bug fixes 🐛</h3>
 
+* `PSWAP` now supports parameter broadcasting.
```

Comment:
```suggestion
* `PSWAP.matrix()` now supports parameter broadcasting.
```

Reply:
It's also in the `eigvals` method. So we could either take the viewpoint of "decomposition was working by accident, now the full thing supports broadcasting" or of "matrix and eigvals now support broadcasting, catching up with decomposition that already featured it :sunglasses:" :D 
I don't have a preference here.

Reply:
I'll vote for the latter for more details. Anyway they are not makeing changelog item too technical yet still focusing on public API.